KUBEVIRTCI_LOCAL_TESTING.md

```diff
@@ -35,6 +35,10 @@ please use `export BYPASS_PMAN_CHANGE_CHECK=true` to bypass provision-manager ch
 export KUBEVIRTCI_PROVISION_CHECK=1
 ```
 
+If `KUBEVIRTCI_PROVISION_CHECK` is not used,
+you can set `KUBEVIRT_CONTAINER_REGISTRY` (default: `quay.io`), `KUBEVIRT_CONTAINER_ORG` (default: `kubevirtci`) and `KUBEVIRT_CONTAINER_SUFFIX` (default: according gocli tag),
```

Comment:
Could we rename the environment variables from `KUBEVIRT` to `KUBEVIRTCI` ?

Reply:
yes and no imho, it is indeed KUBEVIRTCI
but all the other parameters even that they belong to KUBEVIRTCI, have prefix KUBEVIRT, so i don't think we should do just to those ?

Reply:
The rest of the env vars are more related to how kubevirt is deployed and what is deployed alongside it whereas the env vars you are adding are focused on published kubevirtci images. 

I just think that `KUBEVIRT_CONTAINER_REGISTRY` would lead me to think of the image registry used for the kubevirt images.

Reply:
right, done